submissions/pozzitive11/popup/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Hooli-style Popup</title>
+  <link rel="icon" type="image/x-icon" href="/images/favicon.png">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header">
+    <div class="container">
+      <nav class="nav" title="Main navigation">
+        <ul class="list-reset nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Mail
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Images
+            </a>
+          </li>
+          <li class="nav__item nav__item-btn">
+            <input id="popup" class="popup__input" type="checkbox">
+            <label class="nav__btn popup__btn" for="popup">
+              <img class="popup__img" src="./images/popup-button.png" alt="popup">
```

Comment:
Fix styles on focus state 
![image](https://user-images.githubusercontent.com/16196199/180650697-4ed6e94c-f909-4a95-9043-99a5b4113052.png)

Reply:
maybe there are some tips how to fix it? 🥲

Reply:
In the browser by default, only interactive elements have focus, In your case focus goes to `input type="checkbox"` but you hide him using `appearance: none`. Find a way to highlight visible element when input on focus.

Reply:
I want to understand the problem more specifically. That is, the selection of the button should disappear after clicking on it?
![image](https://user-images.githubusercontent.com/79833719/180662761-dc4357ea-69c7-40db-860c-f10d1e75debd.png)

Reply:
or in the sense of adding to the checkbox styles `position: absolute; top: 0px; left: 0px; opacity: 0`?

Reply:
Watch this [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) for a better understanding of how and why we hide checkbox.)

Reply:
Some of your changes brokes styles for focus state. Test your solution before pushing it to the repo.
![image](https://user-images.githubusercontent.com/16196199/180723726-98d8f797-8860-45d0-83ed-ecce3aa287c3.png)